mandatory/2-function-creation.js

```diff
@@ -16,7 +18,18 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num > 100) {
+    return `100%`;
+  }
+
+  num = num.toFixed(2);
```

Comment:
If you fix this line the test will pass. Students found few solutions and posted them on slack. For example one solution is:
```suggestion
  num = parseFloat(num.toFixed(2));
```